app/src/main/res/layout-land/section_title.xml

```diff
@@ -25,14 +25,10 @@
 
     <TextView
       android:id="@+id/section_title_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
+      style="@style/Heading3ViewStart"
       android:layout_marginStart="72dp"
       android:layout_marginTop="28dp"
       android:layout_marginEnd="72dp"
-      android:fontFamily="sans-serif-medium"
-      android:text="@{viewModel.sectionTitleText}"
-      android:textColor="@color/oppiaPrimaryText"
```

Comment:
This also doesn't seem to be defined in the style.

Reply:
Reverted